.github/workflows/ci.yml

```diff
@@ -53,25 +94,32 @@ jobs:
           override: true
 
       - uses: Swatinem/rust-cache@v1
-
+      
       - name: Install cargo-nextest
         uses: taiki-e/install-action@nextest
 
       - name: Run cargo nextest
         uses: actions-rs/cargo@v1
         with:
           command: nextest
-          args: run
+          args: run --package nixpacks --test generate_plan_tests
```

Comment:
Should this also run the module unit tests?

Reply:
Whoops, missed `--lib` argument